app/scripts/views/mixins/floating-placeholder-mixin.js

```diff
@@ -90,6 +92,18 @@ define(function (require, exports, module) {
       this.focusFloatingPlaceholder($inputEl);
     },
 
+    /**
+     * The ridiculous name is to avoid collisions with
+     * functions on consumers.
+     *
+     * @param {Event} event
+     */
+    floatingPlaceholderMixinOnSelect: function (event) {
+      var $inputEl = $(event.currentTarget);
```

Comment:
Should this be `const`? Noticed file uses `var` as well, but I think they could be updated to use `const`. File is not to big, so might be worthwhile to make consistent?